android/build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
 		mavenCentral()
 	}
 	dependencies {
-		classpath 'com.android.tools.build:gradle:7.0.1'
+		classpath 'com.android.tools.build:gradle:7.0.4'
```

Comment:
You also need to update the gradle build tool version in the following file to match. Unfortunately, it's the 1 place we have it duplicated in.
https://github.com/tidev/titanium_mobile/blob/master/android/templates/build/root.build.gradle#L10

You should also update Titanium's [android/package.json](https://github.com/tidev/titanium_mobile/blob/master/android/package.json#L23) file's min SDK build tools to `30.0.3` to match what Google documented [here](https://developer.android.com/studio/releases/gradle-plugin#compatibility-7-1-0). Should look like...
```json
{
	"vendorDependencies": {
		"android build tools": ">=30.0.3 <=31.x"
	}
}
```

Reply:
Thanks for all the infos :+1: I've change the template gradle.

Isn't the "gradle plugin" the one with version 7.0.4? and just the gradle tool the one with 7.4.2? So I can keep the [30.0.2 as a minimum](https://developer.android.com/studio/releases/gradle-plugin#compatibility_2). Their naming convention is strange :rofl:

Reply:
Oh you're right. We only need to update the min if the Android build tool is 7.2.x. Good catch. :stuck_out_tongue: